client/packages/invoices/src/Returns/modals/OutboundReturn/ItemSelector.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react';
+import {
+  ModalRow,
+  ModalLabel,
+  Grid,
+  useTranslation,
+  BasicTextInput,
+} from '@openmsupply-client/common';
+import {
+  ItemStockOnHandFragment,
+  StockItemSearchInput,
+} from '@openmsupply-client/system';
+import { useReturns } from '../../api';
+
+interface ItemSelectorProps {
+  item: ItemStockOnHandFragment | null;
+  disabled: boolean;
+  onChangeItem: (item: ItemStockOnHandFragment) => void;
+}
+
+export const ItemSelector: FC<ItemSelectorProps> = ({
+  item,
+  disabled,
+  onChangeItem,
+}) => {
+  const t = useTranslation();
+
+  const { data } = useReturns.document.outboundReturn();
+
+  const existingItemIds = data?.lines.nodes.map(line => line.itemId);
+
+  return (
+    <>
+      <ModalRow>
+        <ModalLabel
+          label={t('label.item', { count: 1 })}
+          justifyContent="flex-end"
+        />
+        <Grid item flex={1}>
+          <StockItemSearchInput
+            autoFocus={!item}
+            openOnFocus={!item}
+            disabled={disabled}
+            currentItemId={item?.id}
+            onChange={newItem => newItem && onChangeItem(newItem)}
+            extraFilter={
+              disabled
+                ? undefined
+                : item =>
+                    item.availableStockOnHand !== 0 &&
+                    !existingItemIds?.some(id => id === item.id)
+            }
+          />
+        </Grid>
+      </ModalRow>
+      {item && (
+        <ModalRow margin={3}>
```

Comment:
Can we vertically center the label here? The one for the Item row already is, so would be nice if this one matched:
![Screenshot 2024-03-05 at 11 07 16 AM](https://github.com/msupply-foundation/open-msupply/assets/5456533/48f8c764-4bc6-4e1f-b2d7-de28d17056c8)